extensions/macosicons/package.json

```
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "macosicons",
+  "title": "MacOSIcons.com",
```

```suggestion
  "title": "MacOSIcons",
```
Could we use this name instead?